undertow/src/main/java/org/wildfly/extension/undertow/deployment/SharedSessionManagerDeploymentProcessor.java

```diff
@@ -60,12 +64,7 @@ public void deploy(DeploymentPhaseContext phaseContext) throws DeploymentUnitPro
         ServletContainerService servletContainer = deploymentUnit.getAttachment(UndertowAttachments.SERVLET_CONTAINER_SERVICE);
         Integer defaultSessionTimeout = ((sessionConfig != null) && sessionConfig.getSessionTimeoutSet()) ? sessionConfig.getSessionTimeout() : (servletContainer != null) ? servletContainer.getDefaultSessionTimeout() : Integer.valueOf(30);
 
-        ServiceName deploymentServiceName = deploymentUnit.getServiceName();
-
-        ServiceName managerServiceName = deploymentServiceName.append(SharedSessionManagerConfig.SHARED_SESSION_MANAGER_SERVICE_NAME);
-        ServiceName affinityServiceName = deploymentServiceName.append(SharedSessionManagerConfig.SHARED_SESSION_AFFINITY_SERVICE_NAME);
```

Comment:
The `org.jboss.as.web.session.SharedSessionManagerConfig#SHARED_SESSION_AFFINITY_SERVICE_NAME` can now be dropped entirely.